app/dashboards/aspace_collection_dashboard.rb

```diff
@@ -27,17 +28,16 @@ class AspaceCollectionDashboard < Administrate::BaseDashboard
   # SHOW_PAGE_ATTRIBUTES
   # an array of attributes that will be displayed on the model's show page.
   SHOW_PAGE_ATTRIBUTES = %i[
-    id
     name
-    created_at
-    updated_at
+    isilon_assets
   ].freeze
 
   # FORM_ATTRIBUTES
   # an array of attributes that will be displayed
   # on the model's form (`new` and `edit`) pages.
   FORM_ATTRIBUTES = %i[
     name
+    active
```

Comment:
@cdoyle-temple per convo with Michelle I had removed "active" fielld on purpose because decided we did not want to use that currently.

Reply:
@dkinzer My oversight. Should the field be removed from the dashboard altogether? I thought it had got missed since it's still in the available fields